beacon-chain/core/transition/transition.go

```diff
@@ -25,9 +25,9 @@ import (
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
 	"github.com/prysmaticlabs/prysm/v5/monitoring/tracing"
-	"github.com/prysmaticlabs/prysm/v5/monitoring/tracing/trace"
+	prysmTrace "github.com/prysmaticlabs/prysm/v5/monitoring/tracing/trace"
 	"github.com/prysmaticlabs/prysm/v5/runtime/version"
-	goTrace "go.opencensus.io/trace"
+	"go.opentelemetry.io/otel/trace"
```

Comment:
if there are more instances of the prysmTrace maybe just leave that as trace and alias the other one

Reply:
before changing we should make this consistent, maybe I'm missing why we have 2 in the first place

Reply:
I can definitely change the aliases, was just going off what was there originally.
Then in this case it's because we have a function here (`ProcessSlotsCore`) that is receiving the span, so the otel dependency is for the `trace.span` type and the other is for the tracing. I think I could create a spanWrapper type to remove the dependency completely but it seemed to be a bit overkill for the few cases where we have the duplicate.

Reply:
oh i see 🤔 we don't need a spanWrapper that's overkill